plaso/cli/pinfo_tool.py

```diff
@@ -337,6 +337,7 @@ def _PrintSessionsDetails(self, storage):
     """
     for session_number, session in enumerate(storage.GetSessions()):
       session_identifier = uuid.UUID(hex=session.identifier)
+      session_identifier = '{1!s}'.format(session_identifier)
```

Comment:
change 1 to 0 since it is the first argument

Reply:
an alternative is to use the py2to3.UNICODE type macro

Reply:
Done